backend/src/main/java/edu/nus/market/service/AccountServiceImpl.java

```diff
@@ -85,24 +123,24 @@ public ResponseEntity<Object> registerService(Register register){
                 .sameSite("Strict")
                 .build();
             // generate the JWTaccesstoken and send it to the frontend
-        return ResponseEntity.status(HttpStatus.CREATED).header("Set-Cookie", cookie.toString()).body(account);
+        return ResponseEntity.status(HttpStatus.CREATED).header("Cookie", cookie.toString()).body(new RspAccount(account));
         }
     }
 
     @Override
-    public ResponseEntity<Object> deleteAccountService(DelAccReq req) {
+    public ResponseEntity<Object> deleteAccountService(DelAccReq req, int id) {
```

Comment:
Why do you need `DelAccReq` here?
```suggestion
    public ResponseEntity<Object> deleteAccountService(int id) {
```